impeller/geometry/tile_mode.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
`geometry` is primarily used for math utilities. Please place this enum in the entity framework. Perhaps in the `Entity` declaration next to the `BlendMode` enum class.

Reply:
Done.